pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -209,9 +210,13 @@ func (m *manager) RemoveContainer(containerID string) error {
 
 func (m *manager) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitResult {
 	klog.InfoS("Topology Admit Handler")
+	startTime := time.Now()
 
 	metrics.TopologyManagerAdmissionRequestsTotal.Inc()
 	pod := attrs.Pod
 
-	return m.scope.Admit(pod)
+	podAdmitResult := m.scope.Admit(pod)
+	metrics.TopologyManagerAdmissionDuration.Observe(metrics.SinceInSeconds(startTime) * 1000)
```

Comment:
Thanks for changing the code addressing my comments! Unfortunately it seems that using SinceInSeconds() defies a bit the point of changing the unit to milliseconds. Perhaps we can inline the code and it a bit, so it may read like
```
metrics.TopologyManagerAdmissionDuration.Observe(time.Since(startTime).Milliseconds())
```
?

Reply:
Sure, updated as per your suggestion.